src/App.tsx

```diff
@@ -43,33 +60,97 @@ type State = {
   sortType: SortType,
 };
 
-export const App = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
-
-    <button type="button">
-      Sort alphabetically
-    </button>
-
-    <button type="button">
-      Sort by length
-    </button>
-
-    <button type="button">
-      Reverse
-    </button>
-
-    <button type="button">
-      Reset
-    </button>
-
-    <ul className="Goods">
-      <li className="Goods__item">Dumplings</li>
-      <li className="Goods__item">Carrot</li>
-      <li className="Goods__item">Eggs</li>
-      <li className="Goods__item">...</li>
-    </ul>
-  </div>
-);
+export class App extends React.Component <{}, State> {
+  state: State = {
+    isStarted: false,
+    isReversed: false,
+    sortType: 'none',
+  };
+
+  render() {
+    const { isStarted, isReversed, sortType } = this.state;
+
+    const newGoodsFromServer = getReorderedGoods(
+      goodsFromServer,
+      sortType,
+      isReversed,
+    );
+
+    return (
+      <div className="App">
+        {isStarted === false && (
+          <button
+            type="button"
+            onClick={() => {
+              this.setState({
+                isStarted: true,
+              });
+            }}
+          >
+            Start
+          </button>
+        )}
+
+        {isStarted && (
```

Comment:
I think it's better to create separate component for list of goods and it's buttons